index.html

```diff
@@ -0,0 +1,147 @@
+                <ul class="nav-list float-right">
+                    <li><a class="menu-list-item" href="about.html">About</a></li><br>
+                    <li><a class="menu-list-item" href="#about-myself">Program</a></li><br>
+                    <li><a class="menu-list-item" href="#my-form">Join</a></li><br>
+                    <li><a class="menu-list-item" href="#my-form">Sponsor</a></li><br>
+                    <li><a class="menu-list-item" href="#my-form">News</a></li><br>
+                    <li><a href="#" class="mt-3  p-2" id="btn-nav">Company</a></li><br>
+                </ul>
+            </div>
+        </div>
+    </header>
+    <nav class="mobile-nav">
+        <span class="close-button">x</span>
+        <ul class="menu-list-mobile">
+            <li><a class="menu-list-item" href="about.html">About</a></li><br>
+            <li><a class="menu-list-item" href="#main-program">Program</a></li><br>
+            <li><a class="menu-list-item" href="#join">Join</a></li><br>
+            <li><a class="menu-list-item" href="#sponsor">Sponsor</a></li><br>
+            <li><a class="menu-list-item" href="#news">News</a></li><br>
+            <li><a href="#" class="mt-3  p-2" id="btn-nav">BP Launch</a></li><br>
+        </ul>
+    </nav>
+
+    <main class="main-section">
+        <section class="top-bg">
+            <section class="hero container">
+                <br>
+                <h4 class="greetings">"Explore Your Dream Career with Aspire"</h4>
+                <h2 class="hero-title">Get connected <br /> to the experts Globally.</h2>
+                <br>
+            </section>
+
+            <section class="message">
+
+                <div class="container">
+                    <p class="message-p p-3 border border-white">
+                        At Career Guiders, we help young people explore different career options with the help of
+                        industry experts. Our mission is to reduce unemployment in Africa by funding talents and
+                        connecting young people with experts in their field of interest. Join us today and start your
+                        journey to success!
+                    </p>
+                    <h3 class="message-time">24/7 Active Services</h3>
+                    <p>@Rwanda Kigali converntion center</p>
+                </div>
+            </section>
+            <br><br><br>
+        </section>
+        <section class="main-program" id="main-program">
+            <br>
+            <h4 class="text-center section-title-white">Main Program</h4>
+            <hr class="title-line">
+            <div class="program-holder container">
+
+
+            </div>
+            <br>
+            <div class="container text-center">
+                <a href="#" class="join-btn p-3" id="join">Join us now for support</a>
+                <a href="#" class="program-link p-3">See The whole program</a>
+            </div>
+            <br><br>
+        </section>
+
```

Comment:
- [ ] It contains a middle section with a column that includes a section header, a divider, five individual articles, each with an image, title, and description, and a line of text at the bottom. I would suggest you please adjust the spacing by looking at the original design. Also, use the same font. 
- [ ] As you should stick to the design so you cannot change the design so please remove the image background and use the same or similar design. you can change only the content, image but the design. please keep in mind. See the image below and try to stick with it
### Your design
![his](https://github.com/Arsenethierry/capstone-project-aspire-web/assets/81727988/f5bc482a-ad21-4b42-8f3d-1cff27d61862)

### Original design
![origi](https://github.com/Arsenethierry/capstone-project-aspire-web/assets/81727988/6c26721d-8a0f-4dc8-b171-fb13d8eb02cc)